decko/lib/decko/tasks/decko/bootstrap.rake

```diff
@@ -107,15 +110,10 @@ def delete_unwanted_cards
 end
 
 def clear_history
-  Card::Action.delete_old
-  Card::Change.delete_actionless
-
-  conn = ActiveRecord::Base.connection
-  conn.execute("truncate card_acts")
-  conn.execute("truncate sessions")
+  puts "clearing history"
   act = Card::Act.create! actor_id: Card::WagnBotID,
                           card_id: Card::WagnBotID
-  Card::Action.find_each do |action|
-    action.update_attributes!(card_act_id: act.id)
-  end
+  Card::Action.make_current_state_the_initial_state act
+  #conn.execute("truncate card_acts")
+  ActiveRecord::Base.connection.execute("truncate sessions")
```

Comment:
we don't actually use the sessions table any more, right?  so this is harmless but non very meaningful...